### ISSUE TITLE
Additional (/music) entry metadata

### ISSUE THREAD

**Issue by sudojunior** [sudojunior/journey#10]
REF: [Ultimate Guitar](https://ultimate-guitar.com/) has a data entry form that supports multi 'Join phrase' prefixes for each accompning artist added.

![image](https://user-images.githubusercontent.com/8607699/93882882-b6e7c700-fcd8-11ea-987c-f91446e4eabc.png)

More entry fields for live events

![image](https://user-images.githubusercontent.com/8607699/93883131-10e88c80-fcd9-11ea-93c8-00205db03fd9.png)

And a [rubric](https://www.ultimate-guitar.com/contribution/help/rubric) for guitar tabs syntax.

It may be possible to recreate some part of the transcript, but getting lyrics and/or score to synchronise with the video would be put towards the runtime over the build of the site itself.

tl;dr possible but time consuming when deployed on GitHub.

This will be automatically closed to be set as a future task / goal.
